src/language/parser.js

```diff
@@ -9,10 +9,11 @@
 
 import { Source } from './source';
 import { syntaxError } from '../error';
-import { lex, TokenKind, getTokenKindDesc, getTokenDesc } from './lexer';
-import type { Token } from './lexer';
+import { TokenKind } from './lexer';
 import type {
   Name,
+  NamedType,
```

Comment:
can you move to where they were defined before instead of moving them in this rev? They are in the same order as defined in the spec for readability reasons